src/APP/App.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import './App.scss';
+import Header from './partials/Header';
+import OfficesPage from './office/officesPage';
+import DataService from './services/dataService';
+import Animation from '../APP/partials/Animation';
+
+class App extends React.Component{
+
+  constructor(props){
+
+    super(props);
+    this.state = {
+
+      offices: [],
+      view: 'list',
+      loading: true
+
+    }
+  }
+
+  loadOffices(){
+
+     DataService.getData().then(data => {
+
+       this.setState({
+
+         offices: data,
+         loading: false
+
+       })
+     });
+
+  }
+
+  gridViewHandler = () =>{
+
+    this.setState({
+      view: 'grid'
+    })
+
+  }
+
+  listViewHandler = () =>{
+
+    this.setState({
+      view: 'list'
+    })
+  }
+
+  mapViewHandler = () =>{
```

Comment:
gridViewHandler, mapViewHabdler, listViewHandler can be same function with single parameter for the view.

Reply:
also these three arrow functions should be class method.